x-pack/docs/en/watcher/trigger/schedule/cron.asciidoc

```diff
@@ -212,21 +54,7 @@ minute during the weekend:
 // NOTCONSOLE
 
 [[croneval]]
-===== Verifying cron expressions
-
-The {es} {alert-features} provide a
-<<elasticsearch-croneval,`elasticsearch-croneval`>> command line tool
-that you can use to verify that your cron expressions are valid and produce the
-expected results. This tool is provided in the `$ES_HOME/bin` directory.
-
-To verify a cron expression, simply pass it in as a parameter to
-`elasticsearch-croneval`:
-
-[source,bash]
---------------------------------------------------
-bin/elasticsearch-croneval "0 0/1 * * * ?"
```

Comment:
It looks like this section was removed but didn't make it into the new docs? Was this a duplicate? I think we should probably still include a blurb for how to run this.